docs/teamcity.md

```diff
@@ -80,11 +79,11 @@ In addition to the textual output, Sigrid CI also generates a static HTML file t
 
 <img src="images/teamcity-artifacts.png" width="500" />
 
-The information in the HTML report is based on the aforementioned list, though it includes slightly more detail.
+In addition to the textual output, the results also include a link to the Sigrid landing page. 
 
-<img src="images/feedback-report.png" width="600" />
+<img src="images/landing-page.png" width="700" />
 
-Finally, if you want to have more information on the system as a whole, you can also access [Sigrid](http://sigrid-says.com/), which gives you more information on the overall quality of the system, its architecture, and more.
+You can open this URL in order to use Sigrid for interpreting your analysis results. Whether you should use the text output or the Sigrid page is largely down to personal preference: the text output is faster to acces and more concise, while Sigrid allows you to view results in a more visual and interactive way. 
```

Comment:
The link to the Sigrid landing page is technically part of the textual output (contrary to what was mentioned in the old text). I wanted to suggest adding it to the list as another bullet but I think it's clearer to indeed have this text separated from the list. I would however change the text accordingly. I would also move the first sentence of the next paragraph here. The result would be the following:

```
At the end of the textual output there is a link to the Sigrid landing page. You can open this URL in order to use Sigrid for interpreting your analysis results.

<img src="images/landing-page.png" width="700" />

Whether you should use the text output or the Sigrid page is largely down to personal preference: the text output is faster to acces and more concise, while Sigrid allows you to view results in a more visual and interactive way. 
```

(I can't seem to put this as a suggestion directly as GitHub won't allow those on removed lines and even in split mode I can't only select the new and unchanged lines...)

Reply:
Fair point, I'll update it. Same answer also applied to the other occurrences of that text :)